### PR TITLE
Fix parse errors in Fuzer and Hebits

### LINKED ISSUE #7502: [hebits][fuzer] indexes are broken with Parse Error
### Environment

**OS**: Docker

**.Net Runtime**: hotio/jackett:latest

**.Net Version**:

**Jackett Version**: 0.13.446.0

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** Yes

### Description

I get an error when checking hebits and fuzer indexes. 
Exception (fuzer): Object reference not set to an instance of an object.: Parse error

### Logged Error Messages

Jackett.Common.IndexerException: Exception (fuzer): Object reference not set to an instance of an object.
 ---> System.Exception: Parse error
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Jackett.Common.Indexers.Fuzer.performRegularQuery(TorznabQuery query, String hebName) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/Fuzer.cs:line 247
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 830
   at Jackett.Common.Indexers.Fuzer.performRegularQuery(TorznabQuery query, String hebName) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/Fuzer.cs:line 301
   at Jackett.Common.Indexers.Fuzer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/Fuzer.cs:line 181
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 323
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 343
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 803
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 220
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 130
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 26

### PR DESCRIPTION
**Pull request by cadatoiva**
Fixes #7502

### CODE REVIEW

**Comment by ngosang** [comment]
Closed by #7507

File: src/Jackett.Common/Indexers/Fuzer.cs [hunk 1]
```diff
@@ -6,26 +6,28 @@
...
-        public override string[] LegacySiteLinks { get; protected set; } = new string[] {
-            "https://fuzer.me/",
+        public override string[] LegacySiteLinks { get; protected set; } =
+        {
+            "https://fuzer.me/"
         };
 
         private string SearchUrl => SiteLink + "browse.php";
         private string LoginUrl => SiteLink + "login.php";
+        //Unused flagging for removal
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Fuzer.cs]: remove all unused vars you find

File: src/Jackett.Common/Indexers/Hebits.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ namespace Jackett.Common.Indexers
 {
     public class Hebits : BaseWebIndexer
     {
+        //Unused flagging for removal
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Hebits.cs]: remove all unused vars you find

File: src/Jackett.Common/Indexers/Hebits.cs [hunk 3]
```diff
@@ -83,82 +84,48 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
...
                     var dateString = row.QuerySelector("div:last-child").TextContent.Trim();
-                    var pattern = "\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2}";
-                    var match = Regex.Match(dateString, pattern);
+                    var match = Regex.Match(dateString, @"\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}");
                     if (match.Success)
-                    {
                         release.PublishDate = DateTime.ParseExact(match.Value, "yyyy-MM-dd HH:mm:ss", CultureInfo.InvariantCulture);
-                    }
-
                     var sizeStr = row.QuerySelector(".bSize").TextContent;
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Hebits.cs]: add trim, it's not necessary for this case but I think it's a good idea

**Comment by garfield69** [comment]
when I want to work exclusively on a ticket I assign the ticket to myself using the assignees pulldown.
if an open ticket solicit volunteers with multiple assignees, and I want it for myself, then I drop the other assignees from the list.

**Comment by garfield69** [comment]
@ngosang  you posted comments about removing unassigned vars but also approved the ticket.
I take this to mean that you want me to merge the ticket rather than wait for the unassigned vars to be removed.
if this is not the case then please do not approve the ticket, else I cannot judge if I should wait or merge ;-)

**Comment by ngosang** [comment]
@garfield69  You are right. I will try to be more clear in the future.

**Comment by cadatoiva** [comment]
I wasn't sure if I was going to finish before I had to leave for work, so I left it unclaimed. I ended up getting it done, but I'll be more clear about that in the future.

**Comment by cadatoiva** [comment]
I left this open for contributers to push changes, @ngosang so if you would push those changes for me. I won't be able to for several hours.

**Comment by cadatoiva** [comment]
Editing code from a phone is painful, but I got it done.

**Comment by garfield69** [comment]
@cadatoiva  yeah, nah! Don't work from your phone in future, its just too easy to make typo on those things hdbits L112 `var sizeStr = row.QuerySelector(".bSize").Text Content.Trim();` spot the extra blank  :-D
I've fixed it.

**Pull request merged**